main/lsp/requests/completion.cc

```diff
@@ -120,6 +120,12 @@ bool hasAngleBrackets(string_view haystack) {
     return absl::c_any_of(haystack, [](char c) { return c == '<' || c == '>'; });
 }
 
+bool hasSimilarName(const core::GlobalState &gs, core::NameRef name, string_view pattern) {
+    string_view view = name.shortName(gs);
+    auto fnd = view.find(pattern);
+    return fnd != string_view::npos;
+}
```

Comment:
This function was only used in one file: completion.cc, so I figured I'd just move it in there.